src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java

```diff
@@ -159,6 +164,18 @@ private void initialize() {
         	this.setSize(700, 500);
         }
         state = State.IDLE;
+        
+        //  Handle escape key to close the dialog    
+        KeyStroke escape = KeyStroke.getKeyStroke(KeyEvent.VK_ESCAPE, 0, false);
+        AbstractAction escapeAction = new AbstractAction() {
+			private static final long serialVersionUID = 3516424501887406165L;

```

Comment:
Could use spaces for indentation (as surrounding code). Same for some following statements.